src/public/modules/forms/admin/directives/configure-mobile.client.directive.js

```diff
@@ -14,5 +14,45 @@ function configureMobileDirective() {
       name: '=',
       characterLimit: '=',
     },
+    controller: [
+      '$uibModal',
+      '$scope',
+      '$translate',
+      function ($uibModal, $scope, $translate) {
+        // Get support form link from translation json.
+        $translate('LINKS.SUPPORT_FORM_LINK').then((supportFormLink) => {
+          $scope.supportFormLink = supportFormLink
+        })
+
+        $scope.openVerifiedSMSModal = function () {
+          const isTogglingOnVerifiedSms = !$scope.field.isVerifiable
+          $scope.verifiedSMSModal =
+            isTogglingOnVerifiedSms &&
+            $uibModal.open({
```

Comment:
Might want to check console to see if any errors get thrown. I have a suspicion unhandled promise will be thrown.

See: https://github.com/angular-ui/bootstrap/issues/6501, or maybe just pass in `angular.noop` to the result promise handling / catching

Reply:
checked. no errors thrown